front/app/components/admin/InviteUsersWithSeatsModal/InviteUsersWithSeatsModal.test.tsx

```diff
@@ -3,6 +3,53 @@ import { render, screen, userEvent } from 'utils/testUtils/rtl';
 
 import InviteUsersWithSeatsModal from './';
 
+type MockAppConfigurationType = {
+  data: {
+    id: string;
+    attributes: {
+      settings: {
+        core: {
+          maximum_admins_number: number | null;
+          maximum_moderators_number: number | null;
+        };
+      };
+    };
+  };
+};
+
+const mockAppConfiguration: MockAppConfigurationType = {
+  data: {
+    id: '1',
+    attributes: {
+      settings: {
+        core: {
+          maximum_admins_number: 6,
+          maximum_moderators_number: 9,
+        },
+      },
+    },
+  },
+};
```

Comment:
Let's use the data from the api folder as discussed yesterday in the FE CDM from next time? No need to change if you want to make faster progress.

Reply:
We can change later. I also see a potential problem if another test file modifies the object. We probably need a way to enforce that the original object is reset on tests. Not too hard to enforce (🤔 ) but leaving out of this one